src/utils/helpers.ts

```diff
@@ -43,7 +43,9 @@ export function calculateDelay(delay: DelayType, retryCount: number): number {
 }
 
 export function generateQueuePrefixFromPackageName(): string | undefined {
-    return process.env.npm_package_name?.split('/')[0].replace(/[_-]/gi, '.');
+    const parts = process.env.npm_package_name?.split('/');
+
+    return parts?.[parts.length - 1]?.replace(/[_-]/gi, '.');
```

Comment:
Add tests for 4 keys
@goparrot/platform-some-service => some-service
@goparrot/platform-some => some
@goparrot/some-service => some-service
some-service => some-service

Reply:
Yes, that's right